pages/support/[category]/index.js

```diff
@@ -1,37 +1,16 @@
 import { useState, useEffect, useContext } from 'react'
 import styled from 'styled-components'
+import clsx from 'clsx'
```

Comment:
what's this for?

Reply:
I was using it but I'm not at the moment — you're right.
CLSX is a 25 byte utility Ivan introduced me to. It allows us to compose classnames basically. so you can go
```react
<div className={clsx(truthy && ""font-bold", "other classes")>
```

Reply:
I've taken out this import for now but I've kept in clsx to package.json because I think I'm gonna use it / would like to use it more